### PR TITLE
Merge feature/batch_update_members branch into release/v1_3_2 branch

### PR DESCRIPTION
**Pull request by mohdadeebkhan**
## Description
Merge the feature branch into release/v1_3_2
For details please see below PRs.
1) PR #514 
2) PR #519 
3) PR #518 

## Jira Ticket
- N/A

## Technical Approach 
- Created flows for batch member update

## Config Changes
- N/A

## Test Cases
- N/A

## Manual Testing
- N/A

### CODE REVIEW

File: a10_octavia/controller/worker/tasks/a10_network_tasks.py [hunk 1]
```diff
@@ -684,10 +684,22 @@ def __init__(self, *arg, **kwargs):
...
     def _add_vrid_to_list(self, vrid_list, subnet, owner):
         vrid_value = CONF.a10_global.vrid
-        filtered_vrid_list = list(filter(lambda x: x.subnet_id == subnet.id, vrid_list))
-        if not filtered_vrid_list:
-            vrid_list.append(
-                data_models.VRID(
+        if isinstance(subnet, list):
+            subnet_ids = set([s.id for s in subnet])
+            for subnet_id in subnet_ids:
+                filtered_vrid_list = list(filter(lambda x: x.subnet_id == subnet_id, vrid_list))
```
Comment (ytsai-a10) [inline a10_octavia/controller/worker/tasks/a10_network_tasks.py:690]: Is it possible to move line 690~698 to a function, and reuse it after in else case?

File: a10_octavia/controller/worker/tasks/vthunder_tasks.py [hunk 2]
```diff
@@ -853,38 +853,77 @@ class TagInterfaceForMember(TagInterfaceBaseTask):
...
-            LOG.debug("Successfully tagged interface with VLAN id %s for member %s",
-                      str(vlan_id), member.id)
-        except (acos_errors.ACOSException, req_exceptions.ConnectionError) as e:
-            LOG.exception("Failed to tag interface with VLAN id %s for member %s",
+        # Case when batch_update_member is triggered
+        if isinstance(member, list):
+            subnet_list = []
+            for m in member:
+                if m.subnet_id not in subnet_list:
+                    if not m.subnet_id:
```
Comment (ytsai-a10) [inline a10_octavia/controller/worker/tasks/vthunder_tasks.py:861]: Is it possible to move line 861~875 to a function, and reuse it in else case?
And I think if we move this block to a function, then we can use return to replace the continue here in line 865.

File: a10_octavia/controller/worker/tasks/vthunder_tasks.py [hunk 3]
```diff
@@ -853,38 +853,77 @@ class TagInterfaceForMember(TagInterfaceBaseTask):
...
-                if self.is_vlan_deletable():
-                    LOG.warning("Reverting tag interface for member with VLAN id %s", vlan_id)
-                    master_device_id = vthunder.device_network_map[0].vcs_device_id
-                    for device_obj in vthunder.device_network_map:
-                        self.delete_device_vlan(vlan_id, member.subnet_id, vthunder,
-                                                device_id=device_obj.vcs_device_id,
-                                                master_device_id=master_device_id)
+            if isinstance(member, list):
+                for m in member:
+                    if not m.subnet_id:
```
Comment (ytsai-a10) [inline a10_octavia/controller/worker/tasks/vthunder_tasks.py:897]: Is it possible to move line 897~911 to a function, and reuse it in else case?
And I think if we move this block to a function, then we can use return to replace the continue here in line 865.

**Review by ytsai-a10 — CHANGES_REQUESTED** [review]

File: a10_octavia/controller/worker/tasks/vthunder_tasks.py (hunk 3 above)
Comment (NehaKembalkarA10) [reply a10_octavia/controller/worker/tasks/vthunder_tasks.py]: Refactored this in https://github.com/a10networks/a10-octavia/pull/524

File: a10_octavia/controller/worker/tasks/vthunder_tasks.py (hunk 2 above)
Comment (NehaKembalkarA10) [reply a10_octavia/controller/worker/tasks/vthunder_tasks.py]: Refactored this in https://github.com/a10networks/a10-octavia/pull/524

File: a10_octavia/controller/worker/tasks/a10_network_tasks.py (hunk 1 above)
Comment (NehaKembalkarA10) [reply a10_octavia/controller/worker/tasks/a10_network_tasks.py]: Refactored this in https://github.com/a10networks/a10-octavia/pull/524

**Review by ytsai-a10 — APPROVED** [review]

**Pull request merged**